src/components/axes/axis.ts

```diff
@@ -358,26 +357,26 @@ module Plottable {
     }
 
     /**
-     * Gets the size of the gutter in pixels.
-     * The gutter is the extra space between the tick labels and the outer edge of the Axis.
+     * Gets the size of the margin in pixels.
+     * The margin is the extra space between the tick labels and the outer edge of the Axis.
```

Comment:
Can you rewrite the documentation so that it more hints that the margin is indeed the size?

So like
"Gets the margin in pixels."